index.html

```diff
@@ -4,7 +4,7 @@
     <title>Fedora API Specification</title>
     <meta charset='utf-8'>
     <meta name="description" content="Fedora API Specification">
-    <script src='https://www.w3.org/Tools/respec/respec-w3c-common'
+    <script src='http://fedora.info/spec/respec/v14.0.13/respec-w3c-common'
```

Comment:
I initially had a relative link in the same directory as the index.html file. The reason a backed away from that was because browsing to the local index.html in the browser with `file://` did not load the javascript.
But if others are ok with that limitation, I am happy to update.